portfolio/src/main/webapp/style.css

```diff
@@ -65,14 +65,42 @@ ul {
   list-style-type: none;
   padding: 0;
   overflow: auto;
-  border-spacing: 15px;
+  overflow-y:scroll;
 }
 
 li {
   word-break: break-all;
+  margin-top: 10px;
+  border-style: dotted;
+  border-color: pink;
+  display: grid;
+  grid-template-areas: 
+    'header header header header header right'
+    'footer footer footer footer footer footer';
+  grid-auto-columns: 1fr;     
 }
 
-li > p {
+li > div {
   margin-left: 15px;
   margin-right: 15px;
+  padding: 10px 0;
+}
+
+.comment-text {
+  color: rgb(102, 0, 102);
+  font-style: italic;
+  font-weight: initial;
+  grid-area: footer;
+}
+
+.delete-button {
+  margin-top: 10px;
+  width: 26px;
+  grid-area: right;
+  align-content: center;
+  cursor: pointer;
+}
+
+.username-container {
+  grid-area: header;
```

Comment:
As discussed, you are bundling into a PR which is meant to implement delete functionality, some unrelated changes to do with refactoring styling from js to css.

This is likely because you reused the same branch across PRs. While git supports pretty much anything, the typical development workflow is to work with multiple 'feature branches' which track the 'master' branch. Each feature branch ends up in a PR, and gets deleted after the PR gets merged.

This has several advantages:
- Assume we decide moving the styling from js to css was a big mistake. Untangling that from the current branch which also implements a new feature is now loads of work. With 2 separate branches, it's as easy as a git delete branch command.
- PR are super simple and focused. Hence they're super easy and quick to review and understand as each line changed is motivated by the PR description. Right now it's unclear if a change is because of the feature you're implementing or because of some unrelated reason related to some refactoring mentioned in a different PR :)

The downside is merge conflicts are more likely. Sadly merge conflicts are a part of life and it is possible to get good at dealing with them. There's also a more advanced option of stacking PR on top of each other, so that they still get reviewed individually, but reduces the chances of merge conflicts.

Don't worry about this for this PR.